frontend/src/views/MainView.vue

```diff
@@ -70,13 +70,18 @@ export default {
     let token = getAuthToken()
     window.signIn = this.onSignin
     if (token) {
-      await postRequest('account/api/validate_auth_token', token).then(
-        (response) => {
-          if (response.user) {
-            this.userSignedIn({ data: response.user, token: response.token })
-          }
-        },
-      )
+      try {
```

Comment:
Did this section & test just get overwritten somewhere or is this a new change(?)

Reply:
Aight sounds good - looks like its still in master so should be fine